src/components/MultiHopTrade/MultiHopTrade.tsx

```diff
@@ -62,26 +62,57 @@ export const MultiHopTrade = memo(
       exact: true,
     })
 
-    const { chainId, assetSubId, sellChainId, sellAssetSubId, sellAmountCryptoBaseUnit } =
-      match?.params || {}
+    const {
+      chainId,
+      assetSubId,
+      sellChainId,
+      sellAssetSubId,
+      sellAmountCryptoBaseUnit: paramsSellAmountCryptoBaseUnit,
```

Comment:
Renamed since it was confusing what the two (input sell amount vs. sell amount) referred to.